File: output/CodeLlama/avatar/Python/Java_repair/codeforces_624_B.java
import java.util.*;
import java.lang.*;
import java.io.*;
public class codeforces_624_B {
    public static void main (String[] args) {
        Scanner sc = new Scanner ( System.in );
        int n = sc.nextInt ( );
        Long a [] = new Long [ n ];
        for ( int i = 0 ; i < n ; i ++ ) {
            a [ i ] = sc.nextLong ( );
        }
        Arrays.sort ( a , Collections.reverseOrder ( ) );
        long pre = Integer.MAX_VALUE;
        long ans = 0;
        for ( int j = 0 ; j < n ; j ++ ) {
            ans += Math.max ( 0 , Math.min ( pre - 1 , a [ j ] ) );
            pre = Math.max ( 0 , Math.min ( pre - 1 , a [ j ] ) );
        }
        System.out.println ( ans );
    }
}

File: output/StarCoder/avatar/Java/Python/codeforces_624_B.py
import java.util.Arrays ; import java.util.Scanner ; public class codeforces_624_B { public static void main ( String [ ] args ) { Scanner in = new Scanner ( System.in ) ; int n = in.nextInt ( ) ; Long [ ] ans = new Long [ n ] ; for ( int i = 0 ; i < n ; i ++ ) { ans [ i ] = in.nextLong ( ) ; } Arrays.sort ( ans ) ; Long count = ans [ n - 1 ] ; for ( int i = n - 2 ; i >= 0 ; i -- ) { if ( ans [ i ] < ans [ i + 1 ] ) { count += ans [ i ] ; } else { while ( ans [ i ] >= ans [ i + 1 ] ) { ans [ i ] -- ; } if ( ans [ i ] > 0 ) { count += ans [ i ] ; } } } System.out.println ( count ) ; } }